test/controllers/changeset_controller_test.rb

```diff
@@ -1,6 +1,7 @@
 require "test_helper"
 require "changeset_controller"
 
+# rubocop:disable ClassLength
```

Comment:
We normally just change the limit in `.rubocop_tobo.yml` rather than disable the check altogether.